### ISSUE TITLE
Unknown command: run

### ISSUE THREAD

**Issue by mzechmeister** [gnudatalanguage/gdl#1914]
It seems the [`.run`](https://www.nv5geospatialsoftware.com/docs/_run.html) command is not fully implemented. Here an example with `.r`, which should be an abbreviation of `.run` or (`.rnew`?)
```
IDL> .r
- i=1
- end
% Compiled module: $MAIN$.
```
vs.
```
GDL> .r
i=1
end
% Unknown command: r
i=1
end
```

The run command is handy to copy&paste larger and nested code snippets into the command line.
With the option `-t` the source code can be inspected.

I noted that compiling with `.r function.pro` works at least locally; but not when an absolute path is given, like `.r /home/gdl/function.pro` yielding `% Error opening file. File: /home/gdl/function.pro`).

**Comment by jtappin** [comment]
This looks to be largely the same issue as #1868.

**Comment by GillesDuvert** [comment]
Dears, .RUN ( opposed to `.RUN _something_` ) has never been implemented in GDL.

**Comment by mzechmeister** [comment]
@GillesDuvert Are there plans to implement `.run`?
You can consider this as a feature request.
With `_something_` you probably mean the filename of a script. As said it does not work for absolute pathnames.

**Comment by GillesDuvert** [comment]
@mzechmeister indeed, having .r (.run) compile "on the spot" needs a subtle use of the line editor and the ANTLR compiler, not really tricky but clearly not a simple patch.

but, for me .r works with absolute pathnames:
```
GDL> .r /home/gildas/test1.pro 
% Compiled module: TEST1.
```
Except that...  typing 'TEST1' in the above example suffices in most cases if test1.pro is in the !PATH.

Or, did I miss something?

**Comment by mzechmeister** [comment]
I can confirm now that `.r` absolute pathnames works. Now I have version 1.1.

**Comment by GillesDuvert** [comment]
see #1978
